content/en/docs/reference/controllers/volume-persistentvolumeclaim.md

```diff
@@ -0,0 +1,18 @@
+---
+title: PersistentVolumeClaim controller
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+
+The {{< glossary_tooltip text="PersistentVolumeClaim" term_id="persistent-volume-claim" >}}
+controller is part of a set of built-in controllers for storage management.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
```

Comment:
Can you add a `## Controller behavior` section so that the content is consistent with the other pages?